vars/pipelineVegavisorRestartNetwork.groovy

```diff
@@ -327,6 +354,48 @@ void call() {
                     }
                 }
             }
+            stage('provide liquidity') {
+                when {
+                    not {
+                        expression {
+                            params.USE_CHECKPOINT
+                        }
+                    }
+                    expression {
+                        ['restart-network', 'quick-restart-network'].contains(params.ACTION)
+                    }
+                }
+                steps {
+                    dir('devopstools') {
+                        sh 'go run main.go network self-delegate --network' + env.NET_NAME
+                    }
+                }
+            }
+            stage('Create markets'){
+                when {
+                    expression {
+                        params.CREATE_MARKETS
+                    }
+                }
+                options {
+                    timeout(unit: 'SECONDS', time: 30 * 7)
```

Comment:
We need a sleep of 3.5min after proposing markets (ideally move provide-lp inside propose)